npc/quests/quests_moscovia.txt

```diff
@@ -8624,10 +8624,10 @@ OnTouch:
 			mes "- You hear a splashing as -";
 			mes "- something gleaming -";
 			mes "- seems to stare at you!! -";
-			next;
+			close2;
 			specialeffect2 EF_FLASHER;
 			percentheal -100,0;
-			close;
+			end;
```

Comment:
To stay in topic of standardization, could you use parentheses in function calls when writing new scripts or editing old ones? (i.e. `unitkill(getcharid(3));`)